src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/UpdateSessionTemplateDto.kt

```diff
@@ -44,4 +44,7 @@ data class UpdateSessionTemplateDto(
 
   @Schema(description = "list of group references for allowed prisoner incentive levels", required = false)
   val incentiveLevelGroupReferences: List<String>? = null,
+
+  @Schema(description = "If a session is a TAP session.", required = true)
+  val isTAPSession: Boolean,
```

Comment:
This should be in the location group,... may be actually add an exclude param instead